spec/controllers/idv/phone_errors_controller_spec.rb

```diff
@@ -1,86 +1,86 @@
 require 'rails_helper'
 
-shared_examples_for 'an idv phone errors controller action' do
-  describe 'before_actions' do
-    it 'includes before_actions from IdvSession' do
-      expect(subject).to have_actions(:before, :redirect_if_sp_context_needed)
+RSpec.describe Idv::PhoneErrorsController do
+  shared_examples_for 'an idv phone errors controller action' do
```

Comment:
this is one of the files that got indented, recommend `?w=1` to see clearer version of changes here